pkg/scheduler/plugins/drf/drf.go

```diff
@@ -60,12 +64,27 @@ func (drf *drfPlugin) Name() string {
 	return PluginName
 }
 
+// NamespaceOrderEnabled returns the NamespaceOrder for this plugin is enabled in this session or not
+func (drf *drfPlugin) NamespaceOrderEnabled(ssn *framework.Session) bool {
```

Comment:
What is the difference between it and `func (ssn *Session) NamespaceOrderEnabled()`?

Reply:
The `NamespaceOrderEnabled` in plugin just return the value of that plugin. This func return the result of this session and used in `allocate` action.

And in the newest version code, I delete this func and will open a Issue to adjust the behavior when the NamespaceOrder is disabled or other cases.